docs/linting/README.md

```diff
@@ -80,10 +80,18 @@ See following example: https://github.com/demisto/content/blob/fe2bd5cddc6e521e0
 ## Mypy
 Mypy uses type annotations to check code for common errors. It contains type information for many popular libraries (via [typeshed project](https://github.com/python/typeshed)). Additionally, it allows you to define type annotations for your own functions and data structures. Type annotations are fully supported as a language feature in python 3.6 and above. In earlier versions type annotations are provided via the use of comments. 
 
-We run mypy in a relatively aggressive mode so it type checks also functions which are don't contain type definitions. This may sometimes cause extra errors. If you receive errors you can always ignore the line with an inline comment of:
+We run mypy in a relatively aggressive mode so it type checks also functions which don't contain type definitions. This may sometimes cause extra errors. If you receive errors you can always ignore the line with an inline comment of:
 ```python
-# type: ignore
+# type: ignore[<error-name>]
 ``` 
+For example:
+```python
+a = 1
+b = "2"
+a = b  # type: ignore[assignment]
+```
+**Note**: mypy introduced the `ignore[<error-name>]` syntax only in version 0.730. See [blog post](http://mypy-lang.blogspot.com/2019/09/mypy-730-released.html). You may see in the code ignores of the form: `type: ignore` without the `error-name`. This would usually be from old code written before the support for `error-name` ignores. We do not recommend using this ignore style as it ignores all errors and increases the risk of ignoring unexpected serious errors.
```

Comment:
i think directing to their [doc](https://mypy.readthedocs.io/en/latest/error_codes.html) instead of the blog would be better

Reply:
done